src/fsharp/service/ServiceStructure.fs

```diff
@@ -259,7 +260,7 @@ module Structure =
             | SynExpr.LetOrUse (_,_,bindings, body, _) ->
                 parseBindings bindings
                 parseExpr body
-            | SynExpr.Match (seqPointAtBinding,_expr,clauses,_,r) ->
+            | SynExpr.Match (seqPointAtBinding,_expr,clauses,_,r) | SynExpr.MatchBang (seqPointAtBinding, _expr, clauses, _, r) ->
```

Comment:
I’m sure there no such formatting in the whole repository.

Reply:
There is: https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/service/ServiceUntypedParse.fs#L872

Reply:
@cartermp,

From a readability perspective, I think we should prefer the fall through match,  I agree with @vasily-Kirichenko.

It is a certainty that we can using this repo point to an example of every type of formatting style that could possibly exist.  For sure fall through matches with the separate cases per line is the most common, and I think readable format.

Kevin

Reply:
@cartermp,
From a readability perspective, I think we should prefer the fall through match, I agree with @vasily-kirichenko.
It is a certainty that we can using this repo point to an example of every type of formatting style that could possibly exist. For sure fall through matches with the separate cases per line is the most common, and I think readable format.
Kevin